### PR TITLE
adds header for docs,quickstarts and api to react comp

### PR DESCRIPTION
**Pull request by lindseybateman**

### CODE REVIEW

File: docs-header/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+function render () {
+  return dom.div({className: 'flex-outer header_small'},
+    dom.div({className: 'flex-inner'},
+      dom.h1({children: this.props.data})
+    )
+  );
+}
+
+// componentDidMount
+function componentDidMount () {
```
Comment (flenter) [inline docs-header/index.js]: Can go?

Comment (lindseybateman) [inline docs-header/index.js]: yes indeed

File: docs-header/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      case 'docs':
+        data = 'Docs';
+        break;
+      case 'api':
+        data = 'API';
+        break;
+      case 'quickstarts':
+        data = 'Quickstarts';
+        break;
+    }
```
Comment (flenter) [inline docs-header/index.js]: Is this header also being displayed with the 404 page?

If so, we may want to do something like:
```
default:
  data = '404';
  break;
```

File: docs-header/index.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+/**
+ * Create class.
+ */
+
+module.exports = react.createClass({
+  displayName: 'header_small',
+  getDefaultProps: function () {
+    const base = getWindowUrl();
+    var data;
```
Comment (flenter) [inline docs-header/index.js]: Can you initialise this with null?

**Comment by flenter** [comment]
Not sure how much of this javascript stuff is indexed by google, but apart from that... Nice!

File: docs-header/index.js (hunk 3 above)
Comment (lindseybateman) [inline docs-header/index.js]: check

File: docs-header/index.js (hunk 2 above)
Comment (lindseybateman) [inline docs-header/index.js]: 404 has a custom header

**Pull request merged**